web/src/app/core/services/activity.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@angular/core';
+import { Subject, Observable, of, timer } from 'rxjs';
+import { finalize, startWith } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ActivityService {
+
+  private counter: number = 0;
+  private subject: Subject<number> = new Subject();
+
+  // This function will set the progress bar status
+  public setProgressBar(status: boolean): number {
+    switch(status) {
```

Comment:
switch statement should also have default case . Getting error 
![image](https://user-images.githubusercontent.com/10898595/57793082-1f767980-775e-11e9-9b7b-6cea4a655b1d.png)

Reply:
My VSCode is not highlighting this for me, is there something I need to enable?

Reply:
Configuration is already added in tslint. You should also get the same error 
"switch-default": true,

Reply:
Please see this and also refer the screenshot. Its showing the error : tslint(switch-default)
https://github.com/DashboardHub/PipelineDashboard/blob/issue-1223/web/tslint.json#L135

Reply:
👍